python/ray/experimental/client/server.py

```diff
@@ -45,8 +75,10 @@ def serve(connection_str):
 
 
 if __name__ == "__main__":
-    logging.basicConfig()
+    logging.basicConfig(level="INFO")
```

Comment:
Use `ray.utils.setup_logger`?

Reply:
That does an `import ray` -- which we want to avoid. It could be factored out of `ray/utils.py` or we can use a client-server specific logger, either way.